lang/en.yml

```diff
@@ -261,7 +261,7 @@ en:
     EMAIL_RECIPIENT_UNSAVED_FORM: 'You will be able to select from valid form fields after saving this record.'
     EmailFromContent: 'The from address allows you to set who the email comes from. On most servers this will need to be set to an email address on the same domain name as your site. For example on yoursite.com the from address may need to be something@yoursite.com. You can however, set any email address you wish as the reply to address.'
     FROMADDRESS: 'Send email from'
-    FileUploadWarning: 'Files uploaded through this field could be publicly accessible if the exact URL is known'
+    UnrestrictedFileUploadWarning: 'Access to the current upload folder "{path}" is not restricted. Uploaded files will be publicly accessible if the exact URL is known.'
```

Comment:
```suggestion
    UnrestrictedFileUploadWarning: 'Access to the current upload folder "{path}" is unrestricted. Uploaded files will be publicly accessible if the exact URL is known.'
```

Reply:
This language was agreed upon with PUX (and I think 'not restricted' fits the situation better - 'the user has _not_ restricted the access', rather than 'the user has _unrestricted_ the access'.)

Reply:
I think in this sentence structure it is slightly better as "not restricted", putting more emphasis on the "not". Looking at it again I think I would have changed the complete sentence but not going into that. Not really a deal-breaker for me, the benefit of "unrestricted" is consistency with other areas but in this sentence structure "not restricted" seems to read better.